src/Header/HeaderRow.tsx

```
@@ -62,6 +66,8 @@ const HeaderRow = <RecordType extends any>(props: RowProps<RecordType>) => {
 
         return (
           <Cell
+            colIndex={cellIndex}
+            ref={headerCellRef => (headerCellRefs.current[cellIndex] = headerCellRef)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)